EMML/1001-2000/EMML1763.xml

```diff
@@ -953,8 +953,21 @@
 <list>
 
 <item xml:id="a1">
-<locus target="1r"/>
-<desc type="GuestText">Ten lines from the introduction to the Synaxary.</desc>
+<locus target="#1r"/>
+<desc type="GuestText">Ten lines from the introduction to the <ref type="work" corresp="LIT2375Synaxa">Synaxary</ref>.</desc>
+</item>
+
+<item xml:id="a2">
+<locus target="#279r"/>
+<desc type="ScribalSignature"></desc>
+<q xml:lang="gez">ወጸሐፍክዎ፡ ለዝንቱ፡ ድርሳነ፡ አነ፡ መርቆርዮስ፡ ነዳይ፡ ኃጥእ። ጸልዩ፡ ሊተ፡ እለ፡ አንበብክምዎ፡ ለዝንቱ፡ መጽሐፍ። ኵልኩሙ፡ ቅዱሳን፡
```

Comment:
My impression from a superficial look at the manuscript is that this is not really a secondary addition, but a part of the main production unit, probably written by the main hand. If you agree, a better place to encode this information would be in `<colophon>`.
(I looked at it only briefly, if you think it really is an addition, it can stay here)